File: src/main/java/org/example/expert/domain/todo/dto/request/TodoSearchCondition.java
package org.example.expert.domain.todo.dto.request;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class TodoSearchCondition {
    private String titleKeyword;
    private String managerNickname;
    private LocalDate startDate;
    private LocalDate endDate;
}

File: src/main/java/org/example/expert/domain/todo/dto/request/TodoSearchCondition.kt
package org.example.expert.domain.todo.dto.request

import java.time.LocalDate

class TodoSearchCondition (
    val titleKeyword: String? = null,
    val managerNickname: String? = null,
    val startDate: LocalDate? = null,
    val endDate: LocalDate? = null,
    val weather: String? = null
)